src/features/quiz/Results.js

```diff
@@ -1,19 +1,19 @@
 /* eslint-disable */
 
-import React from "react";
+// import React from "react";
 
-export default function Results(props) {
-  return (
-    <>
-      {props.correct ? (
-        <div className="correct-answer">
-          <h1>Correct</h1>
-        </div>
-      ) : (
-        <div className="wrong-answer">
-          <h1>Failed</h1>
-        </div>
-      )}
-    </>
-  );
-}
+// export default function Results(props) {
+//   return (
+//     <>
+//       {props.correct ? (
+//         <div className="correct-answer">
+//           <h1>Correct</h1>
+//         </div>
+//       ) : (
+//         <div className="wrong-answer">
+//           <h1>Failed</h1>
+//         </div>
+//       )}
+//     </>
+//   );
+// }
```

Comment:
The commented out code can be deleted. If someone needs it later, they can find it in the git history.